From 50278dabb69191ed8605164936081a1701256cdd Mon Sep 17 00:00:00 2001
From: Mikhail Naganov <mnaganov@google.com>
Date: Mon, 17 Nov 2014 23:53:31 +0000
Subject: [PATCH] Restore zoom controls in HTMLViewer

WebSettings.setBuiltInZoomControls must be set to 'true' for zoom
controls and pinch-zooming to work. And then we need to call
WebSettings.setDisplayZoomControls(false) to leave only pinch-zoom.

Bug: 18403885
Change-Id: I549fc11673096046212f8c8efaf65824d1000ad7
---
 src/com/android/htmlviewer/HTMLViewerActivity.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/com/android/htmlviewer/HTMLViewerActivity.java b/src/com/android/htmlviewer/HTMLViewerActivity.java
index 0de772d..8a945a9 100644
--- a/src/com/android/htmlviewer/HTMLViewerActivity.java
+++ b/src/com/android/htmlviewer/HTMLViewerActivity.java
@@ -60,6 +60,8 @@ public class HTMLViewerActivity extends Activity {
         WebSettings s = mWebView.getSettings();
         s.setUseWideViewPort(true);
         s.setSupportZoom(true);
+        s.setBuiltInZoomControls(true);
+        s.setDisplayZoomControls(false);
         s.setSavePassword(false);
         s.setSaveFormData(false);
         s.setBlockNetworkLoads(true);
-- 
1.9.1

